src/reducers/updateColumnWidth.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright Schrodinger, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule updateColumnWidth
+ */
+
+'use strict';
+
+import columnWidths from 'columnWidths';
+
+/**
+ * Update our cached col width for a specific index
+ *
+ * NOTE (jordan) This alters state so it shouldn't be called
+ * without state having been cloned first.
+ *
+ * @param {!Object} state
+ * @param {number} columnIdx
+ * @return {number} The new col width
+ */
+function updateColumnWidth(state, columnIdx) {
```

Comment:
identical to [updateRowHeight](https://github.com/schrodinger/fixed-data-table-2/blob/v1.0-beta/src/reducers/updateRowHeight.js#L25)

Why use this? Updating takes O(logN), so we just have this helper to skip doing it if the value is already the same.